### PR TITLE
kogito 1.13 prod nightly enabled

### PR DESCRIPTION
**Pull request by Ginxo**
<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy [hunk 1]
```diff
@@ -88,7 +92,6 @@ pipeline{
                         [\$class: 'StringParameterValue', name: 'KOGITO_DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-kogito-${getNexusFromVersion(KOGITO_CURRENT_PRODUCT_VERSION)}/content-compressed'],
                         [\$class: 'StringParameterValue', name: 'UMB_VERSION', value: '${getUMBFromVersion(KOGITO_CURRENT_PRODUCT_VERSION)}'],
                         [\$class: 'StringParameterValue', name: 'PRODUCT_VERSION', value: '${KOGITO_CURRENT_PRODUCT_VERSION}'],
-                        // [\$class: 'StringParameterValue', name: 'DROOLS_PRODUCT_VERSION', value: '\${DROOLS_CURRENT_PRODUCT_VERSION}'], // Should be uncommented once Current is set for RHPAM 7.13.0
```
Comment (radtriste) [inline job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy:91]: why did you remove this one ? this is targeting main branch or ?

**Review by mareknovotny — APPROVED** [review]
looks good

**Review by radtriste — APPROVED** [review]

**Review by cristianonicolai — APPROVED** [review]
